R/pkg/R/functions.R

```diff
@@ -361,10 +361,13 @@ setMethod("column",
 #'
 #' @rdname corr
 #' @name corr
-#' @family math functions
+#' @family aggregate functions
 #' @export
 #' @aliases corr,Column-method
-#' @examples \dontrun{corr(df$c, df$d)}
+#' @examples
+#' \dontrun{
+#' df <- createDataFrame(cbind(model = rownames(mtcars), mtcars))
```

Comment:
great - I know we talk about it, but we might consider getting all examples on the Rd into one `\dontrun` block again. as of now it's very hard to review new PR without knowing whether a newline is needed or not...